docs/gauge_agg.md

```diff
@@ -0,0 +1,76 @@
+# Gauge Aggregates [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
+
+A gauge is a metric similar to a counter, with the primary difference being
+that it measures a value that varies up and down over time, rather than an
+ever-increasing COUNT of the number of times something happened.
+Examples include resource utilization metrics, precipitation levels,
```

Comment:
Do we want a specific example of each and a comparison? This is a good explanation, but I think an illustration like the number of visitors to a website is a counter, because x and this is a gauge because why and this is how you can tell the difference.  It feels like that would help but I might be wrong.